StoreApp/Controllers/AccountController.cs

```diff
@@ -212,18 +213,21 @@ public ActionResult EditUserAdmin(UserViewModel user)
         {
             if ((Session["Role"] as string).Equals(UserRoles.admin.ToString()))
```

Comment:
I propose to change 
**admin** to **Admin**
**user** to **User**
It looks better

Reply:
fixed